src/i18n/en-US.json

```diff
@@ -552,22 +550,26 @@
   "modalAccountRemoveDevicePlaceholder": "Password",
   "modalAcknowledgeAction": "Ok",
   "modalAcknowledgeHeadline": "Something went wrong",
+  "modalAppLockSetupSpecial": "A special character",
+  "modalAppLockSetupLong": "At least {{minPasswordLength}} charcters long",
+  "modalAppLockSetupLower": "A lowercase letter",
+  "modalAppLockSetupUppercase": "An uppercase letter",
+  "modalAppLockSetupDigit": "A digit",
   "modalAppLockForgotGoBackButton": "Go back",
   "modalAppLockForgotMessage": "The data stored on this device can only be accessed with your app lock passphrase. If you have forgotten your passphrase, you can reset this client.",
   "modalAppLockForgotTitle": "Forgot your app lock passphrase?",
   "modalAppLockForgotWipeCTA": "Reset this client",
   "modalAppLockLockedError": "Wrong passphrase",
-  "modalAppLockLockedForgotCTA": "Forgot passphrase?",
-  "modalAppLockLockedMessage": "Please enter your app lock passphrase.",
-  "modalAppLockLockedPlaceholder": "Passphrase",
-  "modalAppLockLockedTitle": "App locked due to inactivity",
+  "modalAppLockLockedForgotCTA": "Access as new device",
+  "modalAppLockPasscode": "Passcode",
+  "modalAppLockLockedTitle": "Enter passcode to unlock Wire",
   "modalAppLockLockedUnlockButton": "Unlock",
   "modalAppLockSetupAcceptButton": "Set passphrase",
-  "modalAppLockSetupInfo": "Use at least {{minPasswordLength}} characters, with one lowercase letter, one capital letter, a number, and a special character.",
   "modalAppLockSetupMessage": "The app will lock itself after a certain time of inactivity.[br]To unlock the app you need to enter this passphrase.[br]Make sure to remember this passphrase as there is no way to recover it.",
-  "modalAppLockSetupPlaceholder": "Passphrase",
+  "modalAppLockSetupChangeMessage": "Your organization needs to lock your app when Wire is not in use to keep the team safe .[br]Create a passlock to unlock Wire. Please, remeber it, as it can not be recovered.",
```

Comment:
Ah here are also some mentions of `Wire` that needs to be replaced with `BRAND_NAME`